extensions/src/ACRE2TS/TS3Client.cpp

```diff
@@ -117,6 +116,28 @@ acre::Result CTS3Client::exPersistVersion( void ) {
     return acre::Result::error;
 }
 
+acre::Result CTS3Client::setSelfVariable(char* data) {
```

Comment:
I believe this is constant
```suggestion
acre::Result CTS3Client::setSelfVariable(const char *const data) {
```